src/Instrumentation/Laravel/src/LaravelInstrumentation.php

```diff
@@ -92,20 +94,54 @@ public static function register(): void
                 $span->end();
             }
         );
+
+        hook(
+            ConsoleKernel::class,
+            'call',
+            pre: static function (ConsoleKernel $kernel, array $params, string $class, string $function, ?string $filename, ?int $lineno) use ($instrumentation) {
+                /** @psalm-suppress ArgumentTypeCoercion */
+                $builder = $instrumentation->tracer()
+                    ->spanBuilder(sprintf('Console %s', $params[0] ?? 'unknown'))
+                    ->setSpanKind(SpanKind::KIND_PRODUCER)
```

Comment:
I'm not sure if this is correct or not?

Reply:
Good question. Nothing in https://github.com/open-telemetry/opentelemetry-specification/blob/v1.24.0/specification/trace/api.md#spankind seems to quite fit for a console application - `producer` seems like a good choice.